lib/smrt_client.py

```
@@ -0,0 +1,113 @@
+import base64
+from datetime import datetime, timezone
+
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+CLIENT_KEY = "6NjRXBcFfLZOwHc0Xlidiz4ywcsa"
+CLIENT_SECRET = "KMLz5g7fbmx8RVFKKdu0NOrJic4a"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "CLIENT_SECRET"

(S105)